easybuild/easyconfigs/b/Boost/Boost-1.61.0-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,26 @@
+name = 'Boost'
+version = '1.61.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.boost.org/'
+description = "Boost provides free peer-reviewed portable C++ source libraries."
+
+toolchain = {'version': '2016b', 'name': 'foss'}
```

Comment:
@verdurin please fix to the usual order (this is a bug in the dumping of easyconfigs that is not resolved yet)